images/node/Dockerfile

```diff
@@ -15,15 +15,13 @@ COPY system-container/manifest.json system-container/config.json.template system
 
 RUN INSTALL_PKGS=" \
       origin-hyperkube origin-node origin-sdn-ovs \
-      socat ethtool device-mapper iptables nmap-ncat e2fsprogs \
-      xfsprogs device-mapper-persistent-data ceph-common \
+      socat ethtool device-mapper iptables nmap-ncat e2fsprogs ceph-common \
+      xfsprogs device-mapper-persistent-data\
       libmnl libnetfilter_conntrack conntrack-tools \
       libnfnetlink iproute bridge-utils procps-ng openssl \
       binutils xz kmod-libs kmod sysvinit-tools device-mapper-libs dbus \
       iscsi-initiator-utils \
       " && \
-    yum install -y centos-release-ceph-luminous && \
-    rpm -V centos-release-ceph-luminous && \
```

Comment:
You need to still install this here if it isn’t already.  You should remove all changes to this file.